src/dmd/backend/global.h

```diff
@@ -483,7 +483,7 @@ void rtlsym_reset();
 void rtlsym_term();
 
 // compress.c
-char *id_compress(char *id, int idlen, size_t *plen);
+extern "C" char *id_compress(char *id, int idlen, size_t *plen);
```

Comment:
Need `{ }` after the `extern "C"`

Reply:
BTW there are a few other files that don't use the `{ }`:

```
dmd/backend/os.c
646:extern "C" void * __cdecl _osfhnd[];

dmd/backend/type.h
200:extern "C" type *type_static_array(targ_size_t dim, type *tnext);
203:extern "C" type *type_function(tym_t tyf, type **ptypes, size_t nparams, bool variadic, type *tret);

tests/cxxfrontend.c
61:extern "C" int rt_init();
62:extern "C" void gc_disable();
82:extern "C" int rt_term();
83:extern "C" void gc_enable();
```